index.html

```diff
@@ -0,0 +1,171 @@
+              <div class="bio">
+                Sed tortor metus, aliquet at libero a, tempus euismod est. Class
+                aptent taciti sociosqu ad litora torquent per.
+              </div>
+              <!-- Buttons for linkedin, twitter, and anything else -->
+              <a href=""></a>
+            </div>
+
+            <div class="person-card">
+              <div class="name">Jarrad Miller</div>
+              <div class="position">Back End Architect</div>
+              <div class="bio">
+                Sed tortor metus, aliquet at libero a, tempus euismod est. Class
+                aptent taciti sociosqu ad litora torquent per.
+              </div>
+              <!-- Buttons for linkedin, twitter, and anything else -->
+              <a href=""></a>
+            </div>
+
+            <div class="person-card">
+              <div class="name">Rory Hennessey</div>
+              <div class="position">UI Developer 1</div>
+              <div class="bio">
+                Sed tortor metus, aliquet at libero a, tempus euismod est. Class
+                aptent taciti sociosqu ad litora torquent per.
+              </div>
+              <!-- Buttons for linkedin, twitter, and anything else -->
+              <a href=""></a>
+            </div>
+
+            <div class="person-card">
+              <div class="name">Brian Illes</div>
+              <div class="position">UI Developer 2</div>
+              <div class="bio">
+                Sed tortor metus, aliquet at libero a, tempus euismod est. Class
+                aptent taciti sociosqu ad litora torquent per.
+              </div>
+              <!-- Buttons for linkedin, twitter, and anything else -->
+              <a href=""></a>
+            </div>
+
+            <div class="person-card">
+              <div class="name">Adam Mathieson</div>
+              <div class="position">UI Developer 3</div>
+              <div class="bio">
+                Sed tortor metus, aliquet at libero a, tempus euismod est. Class
+                aptent taciti sociosqu ad litora torquent per.
+              </div>
+              <!-- Buttons for linkedin, twitter, and anything else -->
+              <a href=""></a>
+            </div>
+          </div>
+
+          <div class="contact" id="contact">
+            <h2>Contact</h2>
+            <p>Reach us through the following mediums:</p>
+            <div class="contact-app-links">
+              <!-- buttons for our Company email(opens native email app), our Twitter -->
+            </div>
+          </div>
+
+          <div class="subscribe">
+            <h2>Get updates about Tabless Thursday</h2>
+            <form action="">
+              <input type="email" placeholder="Enter your email" />
+              <a href="#">Subscribe</a>
+            </form>
+          </div>
+        </section>
+      </div>
+    </div>
+
+    <footer>
+      <p>Copyright Tabless Thursday 2019</p>
+    </footer>
+  </body>
```

Comment:
Love this HTML structure - the card based interface is wonderful, clean , and a personal favorite of mine.  This will be much easier to do with React's component-based rendering, but wow, I love your implementation of it with pure HTML markup.